doc/cabal-commands.rst

```diff
@@ -191,7 +191,7 @@ Target Forms
 
 A cabal command target can take any of the following forms:
 
--  A package target: ``package``, which specifies that all enabled
+-  A package target: ``[pkg:]package``, which specifies that all enabled
```

Comment:
without reading the whole thing, do you think is clear with the new version you can use `cabal install my-package` if there is no ambiguities?

Reply:
The other entries show "optional syntax" in the same way as this. As to whether that's clear... I managed to understand it, but if we change it we should change the whole section so it's consistent.